up42/__init__.py

```diff
@@ -56,7 +56,6 @@
 from up42.tasking import Tasking
 from up42.tools import get_example_aoi, read_vector_file
 from up42.utils import get_up42_py_version
-from up42.viztools import draw_aoi
```

Comment:
We need to clean up mypy.ini and pylintrc as well

Reply:
good catch!